### PR TITLE
Make it easier to run uttu locally

### PR DESCRIPTION
**Pull request by christianmosveen**
Added a recipe for running uttu locally to the README.

Basically created a Spring application properties file for running locally, and made the spring-boot-maven-plugin use the `local` profile so that it picks up those application properties (and included how to do the same for running in IntelliJ in the README).

### CODE REVIEW

**Comment by testower** [comment]
Great! I personally just run postgresql from docker - maybe worth a mention?

**Comment by FredrikMeyer** [comment]
Nice! 

I got Docker to work like this:
```
docker run --name=uttu -d -e POSTGRES_USER=uttu -e POSTGRES_PASS=uttu -e POSTGRES_DBNAME=uttu -e ALLOW_IP_RANGE=0.0.0.0/0 -p 5432:5432 -v db_local:/var/lib/postgresql --restart=always kartoza/postgis:9.6-2.4
```

Then I added a script to run the migrations:
```bash
#!/bin/bash  -e

for f in ./db/migration/V1__Base_version.sql \
           ./db/migration/V2__Add_Export_filename.sql \
           ./db/migration/V3__Remove_unique_order_constraints.sql \
           ./db/migration/V4__Increased_text_column_length.sql \
           ./db/migration/V5__Increased_booking_note_text_column_length.sql \
           ./import.sql
do
  echo "Running migration for ${f}"
  PGPASSWORD=uttu psql -U uttu -h localhost -p 5432 -f $f
done
```

I also had to the following two lines in `application-local.properties`:
```
spring.datasource.username=uttu
spring.datasource.password=uttu
```

I can make a PR describing this later (or perhaps a PR into this branch?).

File: README.md [hunk 1]
```diff
@@ -1,13 +1,57 @@
...
+You need Java 11, Maven 3 or higher, and a `settings.xml`-file that gives you access to the internal repo (ask a friend).
+
+### Database
+
+#### Via local installation of database
+Install Postgres, either via [brew](https://gist.github.com/ibraheem4/ce5ccd3e4d7a65589ce84f2a3b7c23a3), [postgresapp.com](http://postgresapp.com/),
+or [postgresql.org](https://www.postgresql.org/download/).
+
+For brew install, also install PostGIS: `brew install postgis` (should be included in Postgres.app and EnterpriseDB Postgres installations).
+
```
Comment (simejo) [inline README.md]:
```suggestion
Create folder for database `mkdir db`
Initialise database `initdb ./db`
Start db service: `pg_ctl -D ./db/ -l logfile start`
Create db with your username: `createdb `whoami``
```

File: README.md [hunk 2]
```diff
@@ -1,13 +1,57 @@
...
+Create uttu database: `createdb uttu`
+
+Create uttu user: `createuser -s uttu` (you might also have to create a `postgres` user)
+
+Run the [migrations](src/main/resources/db.migration) in order.
```
Comment (simejo) [inline README.md]:
```suggestion
Run the [script](./src/main/resources/db_init.sh)
```

File: README.md [hunk 3]
```diff
@@ -1,13 +1,57 @@
...
+
+Run the initial [data import](src/main/resources/import.sql)
```
Comment (simejo) [inline README.md]:
```suggestion
```

**Pull request merged**